dataunits.rst

```diff
@@ -9,6 +9,12 @@ DataUnit Reference
 DataUnit
 --------
 
+.. warning::
+
+  The Python representation of :ref:`DataUnits <DataUnit>` is likely to change in the new preflight design. Specifically instances of :py:class:`DataUnit` are likely to represent Registry database tables as opposed to rows in such tables.
```

Comment:
Should add that this means that we just intend to use `dict`s to represent the rows of `DataUnit` tables.

Could you also add a similar disclaimer to the Preflight operations section, saying that it's under construction?